client/src/components/payment/payment-modal.vue

```diff
@@ -1,11 +1,12 @@
 <template>
-  <v-dialog max-width="40%" :value="isVisible" persistent>
-    <v-card class="pa-9">
-      <v-card-text class="h3">
-        Confirm Name Request
+  <v-dialog max-width="50%" :value="isVisible" persistent>
+    <v-card>
+      <v-card-title class="d-flex justify-space-between">
+        <div>Confirm Name Request</div>
```

Comment:
Just curious why a Div over a span here? Feels more like an inline element opposed to a block/section.. though i could be wrong!

Reply:
Span works as well. Flexbox doesn't seem to care.

Reply:
Cool!